examples/seismic/viscoacoustic/operators.py

```diff
@@ -642,16 +636,14 @@ def BornOperator(model, geometry, space_order=4, kernel='sls', time_order=2, **k
     eq_kernel = kernels[kernel]
     eqn1 = eq_kernel(model, geometry, p, r=rp, **kwargs)
 
-    s = model.grid.stepping_dim.spacing
-
     if time_order == 1:
         dv = VectorTimeFunction(name="dv", grid=model.grid,
                                 time_order=time_order, space_order=space_order)
         kwargs.update({'v': dv})
 
-        q = -dm * (p.forward - p) / s
+        q = -dm * p.dt
     else:
-        q = -dm * (p.forward - 2 * p + p.backward) / (s**2)
+        q = dm * p.dt2
```

Comment:
was this broken?

Reply:
No it's the same thing just nicer with the derivatve